jts-test-runner/src/input.rs

```diff
@@ -229,3 +229,68 @@ where
     String::deserialize(deserializer)
         .and_then(|str| T::from_str(&str).map_err(serde::de::Error::custom))
 }
+
+pub fn deserialize_point_from_wkt<'de, D>(
+    deserializer: D,
+) -> std::result::Result<Option<Point<f64>>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    Ok(wkt::deserialize_point::<D, f64>(deserializer)?.map(to_point))
+}
+
+pub fn deserialize_from_wkt<'de, D>(deserializer: D) -> std::result::Result<Geometry<f64>, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    return Ok(convert(wkt::deserialize_geometry::<D, f64>(deserializer)?));
+}
+
+fn convert(itm: geo_types::Geometry<f64>) -> geo::GeometryTZM<f64, NoValue, NoValue> {
+    match itm {
+        geo_types::Geometry::Point(v) => geo::Geometry::Point(Point::<f64>::new(v.x(), v.y())),
+        geo_types::Geometry::LineString(v) => geo::Geometry::LineString(to_line_string(v)),
+        geo_types::Geometry::Polygon(v) => geo::Geometry::Polygon(to_polygon(v)),
+        geo_types::Geometry::MultiPoint(geo_types::MultiPoint(v)) => {
+            geo::Geometry::MultiPoint(geo::MultiPoint::new(v.into_iter().map(to_point).collect()))
+        }
+        geo_types::Geometry::MultiLineString(geo_types::MultiLineString(v)) => {
+            geo::Geometry::MultiLineString(geo::MultiLineString::new(
+                v.into_iter().map(to_line_string).collect(),
+            ))
+        }
+        geo_types::Geometry::MultiPolygon(geo_types::MultiPolygon(v)) => {
+            geo::Geometry::MultiPolygon(geo::MultiPolygon::new(
+                v.into_iter().map(to_polygon).collect(),
+            ))
+        }
+        geo_types::Geometry::GeometryCollection(geo_types::GeometryCollection(v)) => {
+            geo::Geometry::GeometryCollection(geo::GeometryCollection::new_from(
+                v.into_iter().map(convert).collect(),
+            ))
+        }
+        geo_types::Geometry::Line(v) => todo!("{v:?}"),
+        geo_types::Geometry::Rect(v) => todo!("{v:?}"),
+        geo_types::Geometry::Triangle(v) => todo!("{v:?}"),
```

Comment:
Instead of `todo!` should we put `unexpected!` since we don't ever expect these lines to get hit?

Reply:
actually @lnicola has suggested i change this part entirely (I'm not yet certain how it would look like, some more feedback is welcome). Otherwise yes, agree, will fix it in interim.